sections/06_discussion.md

```diff
@@ -123,93 +119,41 @@ quantifying the change in the output. Umarov et al.
 windows where the sequence within each sliding window was substituted with a
 random sequence. Kelley et al. [@tag:Kelley2016_basset] inserted known
 protein-binding motifs into the centers of sequences and assessed the change in
-predicted accessibility.
-
-Ribeiro et al. [@tag:Ribeiro2016_lime] introduced LIME, which constructs a linear
+predicted accessibility. Ribeiro et al. [@tag:Ribeiro2016_lime] introduced LIME, which constructs a linear
 model to locally approximate the output of the network on perturbed versions of
 the input and assigns importance scores accordingly. For analyzing images,
 Zeiler and Fergus [@tag:Zeiler2013_visualizing] applied constant-value masks to
-different input patches and studied the changes in the activations of later
-layers. As an alternative to using masks, which can look artificial compared to
-typical values for an input patch, Zintgraf et al.
-[@tag:Zintgraf2017_visualizing] proposed a novel strategy based on marginalizing
-over plausible values of an input patch to more accurately estimate its
-contribution.
+different input patches. More recently, marginalizing over the plausible values
+of an input has been suggest as a way to more accurately estimate contributions
+[@tag:Zintgraf2017_visualizing].
 
 A common drawback to perturbation-based approaches is computational efficiency:
 each perturbed version of an input requires a separate forward propagation
 through the network to compute the output. As noted by Shrikumar et al.
 [@tag:Shrikumar2017_learning], such methods may also underestimate the impact of
 features that have saturated their contribution to the output, as can happen
 when multiple redundant features are present.
-
 To reduce the computational overhead of perturbation-based approaches, Fong and
 Vedaldi [@tag:Fong2017_perturb] solve an optimization problem using gradient
 descent to discover a minimal subset of inputs to perturb in order to decrease
-the predicted probability of a selected class. When tested on image data, their
-method took about 300 iterations to converge, compared to the ~5000 iterations
-used by LIME. One drawback of this approach is that the use of gradient descent
+the predicted probability of a selected class. Their method converged in many
+fewer iterations but
 requires the perturbation to have a differentiable form.
 
-##### Backpropagation-based approaches
-
-A second strategy for addressing the computational inefficiency of
-perturbation-based approaches is to propagate an important signal from a target
-output neuron backwards through the layers to the input layer in a single
-backpropagation-like pass. A classic example of this is calculating the gradients
+Backpropagation-based methods, in which the signal from a target output neuron is propagated backwards to the input layer, are another way to interpret deep networks that sidestep inefficiencies of the perturbastion-basd methods.
```

Comment:
I think the references (or nearly all of them) were preserved but the section was dramatically shortened.